### DISCUSSION TITLE
Several nivo charts of the latest version 0.79.0 don't work with my react v16.14.0 project

### DISCUSSION THREAD

**Discussion by mariela-plaza** [plouc/nivo#2067] [Q&A]
I was thinking of using several nivo charts for a feature I was asked to implement in the react project I am working on.
The react version of said project is 16.14.0. To try out some ideas before I implemented them in the real project, I started out a pet project in which I was able to implement the Heatmap Chart and integrate it with our rest API.

The issue is that the pet's project version of react is v18, a detail I didn't take into account until now.
What are your suggestions in this regard? 

I also tried working with previous versions of the library (version 0.73) but there doesn't seem to be any documentation about the Charts in this version and the Heatmap Chart, which is the one we are most interested in, has changed a lot between version 0.73 and version 0.79.

I appreciate your feedback and responses, have a great day!

**Comment by mariela-plaza** [comment]
The issue was solved, it was a miscalculation on my part.

Sorry guys! Keep up the great work!

**Reply by JanderSilv** [reply]
Hey @mariela-plaza, what was this "miscalculation" 😄? Trying to use at react v16 project as well.
